syntax = "proto3";

package pl.stirling.testing;

option go_package = "pixielabs.ai/pixielabs/src/stirling/testing/proto;greetpb";

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
